cmd/audit.go

```diff
@@ -74,18 +75,19 @@ var auditCmd = &cobra.Command{
 
 		sigsYamlPath, err := utils.GetSigsYamlFile(pwd)
 		if err != nil {
-			panic(fmt.Errorf("ERROR: unable to find sigs.yaml file: %w", err))
+			return err
 		}
 		context, err := utils.GetSigsYaml(sigsYamlPath)
 		if err != nil {
-			panic(fmt.Errorf("ERROR: parsing file: %s - %w", sigsYamlPath, err))
+			return err
```

Comment:
No strong opinion on retaining the format string, will defer to @\nikhita, @\palnabarun and @\dims

Reply:
This function expected a return statement at the end, so I've returned `err` that was last assigned, is there a better way to take care of that?